plugins/presentation/src/components/internal/Properties.svelte

```diff
@@ -47,17 +84,41 @@
             </td>
             <td>
               <div class="edit">
-                <AttributeEditor attribute={attr} bind:value={object[attr.key]} />
+                {#if attr.presenter === 'component:presentation.RefPresenter'}
```

Comment:
As discussed we should construct an Editor here and do not worry about how it will be implemented. So in case of Ref it could be a drop down with search, etc. 

AttributeEditor => should construct and editing control. Right now we have only presenters, but I think we should have presenter to display and editor to edit. Since they could be a quite different controls.

Reply:
Would have UX mixin "editor" attribute like "presenter"?  Is obsolete "editable" field in all presenter?